src/plugins/intel_gpu/tests/common/opencl_helper_instance.hpp

```diff
@@ -23,8 +23,7 @@ struct OpenCL {
     bool _supports_usm;
```

Comment:
moved this class from `tests/unit_tests/utils` to `tests/common/utils` in order to reuse it in sanity tests for the GPU integration